index.d.ts

```diff
@@ -0,0 +1,238 @@
+     */
+    zeroOrMore: SuperExpressive;
+
+    /**
+     * Assert that the proceeding element may not be matched, or may be matched multiple times, but as few times as possible.
+     */
+    zeroOrMoreLazy: SuperExpressive;
+
+    /**
+     * Assert that the proceeding element may be matched once, or may be matched multiple times.
+     */
+    oneOrMore: SuperExpressive;
+
+    /**
+     * Uses the g flag on the regular expression, which indicates that it should match multiple values when run on a string.
+     */
+    oneOrMoreLazy: SuperExpressive;
+
+    /**
+     * Assert that the proceeding element will be matched exactly `n` times.
+     * @param {number} n
+     */
+    exactly(n: number): SuperExpressive;
+
+    /**
+     * Assert that the proceeding element will be matched at least `n` times.
+     */
+    atLeast(n: number): SuperExpressive;
+
+    /**
+     * Assert that the proceeding element will be matched somewhere between `x` and `y` times.
+     */
+    between(x: number, y: number): SuperExpressive;
+
+    /**
+     * Assert that the proceeding element will be matched somewhere between `x` and `y` times, but as few times as possible.
+     */
+    betweenLazy(x: number, y: number): SuperExpressive;
+
+    /**
+     * Assert the start of input, or the start of a line when `.lineByLine` is used.
+     */
+    startOfInput: SuperExpressive;
+
+    /**
+     * Assert the end of input, or the end of a line when `.lineByLine` is used.
+     */
+    endOfInput: SuperExpressive;
+
+    /**
+     * Matches any of the characters in the provided string `chars`.
+     */
+    anyOfChars(chars: string): SuperExpressive;
+
+    /**
+     * Matches any character, except any of those in the provided string `chars`.
+     */
+    anythingButChars(chars: string): SuperExpressive;
+
+    /**
+     * Matches any string the same length as `str`, except the characters sequentially defined in `str`
+     */
+    anythingButString(str: string): SuperExpressive;
+
+    /**
+     * Matches any character, except those that would be captured by the `.range` specified by `a` and `b`.
+     */
+    anythingButRange(a: number, b: number): SuperExpressive;
+
+    /**
+     * Matches the exact string `s`.
+     */
+    string(s: string): SuperExpressive;
+
+    /**
+     * Matches the exact character `c`.
+     */
+    char(c: string): SuperExpressive;
+
+    /**
+     * Matches any character that falls between `a` and `b`. Ordering is defined by a characters ASCII or unicode value.
+     */
+    range(a: string, b: string): SuperExpressive;
+
+    /**
+     * Outputs a string representation of the regular expression that this SuperExpression models.
+     */
+    toRegexString(): string;
+    
+    /**
+     * Outputs the regular expression that this SuperExpression models.
+     */
+    toRegex(): RegExp;
+}
+
+/**
+ * Creates an instance of SuperExpressive.
+ */
+declare function SuperExpressive(): SuperExpressive;
+
+export default SuperExpressive;
```

Comment:
```suggestion
export = SuperExpressive;
```

Because it's defined as `module.exports = SuperExpressive` and not as an ES6 default export